api/app/controllers/spree/api/v2/platform/google_export_options_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Spree
+  module Api
+    module V2
+      module Platform
+        class GoogleExportOptionsController < ResourceController
+          def show
+            @options = GoogleExportOption.find_by store: params[:store_id]
+
+            send_data @options.export, filename: 'products.rss', type: 'text/xml'
```

Comment:
let's say we have laarge number of products to export, I'm not sure if HTTP request will handle that, thinking about some possible timeout. Did you think about some background job for that ? 🤔

Reply:
I don't think I understand how jobs work in rails that well to be honest, but I don't think it could be applied here without bigger changes? The way uploading rss feed for google works is you register a url and google fetches the feed on a set schedule (https://support.google.com/merchants/answer/1219255?hl=en&ref_topic=3404780)
It would probably work with google's api though, becouse than you can upload products one by one or in batches. (https://developers.google.com/shopping-content/guides/making-requests)
I'm not sure if what I'm saying makes sense.

Reply:
I asked about background job because I suspected that generating big numbers of data in controller could end some timeout etc. BUT, I saw in google documentation that you are able to setup `export` schedule, which is cool, also they allowing bigs files ( 4GB ) so here we are good 👌🏻